capi/capi.go

```diff
@@ -3,54 +3,54 @@ package main
 import "C"
 
 import (
-    "gitlab.com/virtblocks/virtblocks/devices"
-    "gitlab.com/virtblocks/virtblocks/util"
+	"github.com/virtblocks/virtblocks/pkg/devices"
```

Comment:
According to the offline chat this would now be `github.com`.

Reply:
Of course :)

Reply:
Now a separate commit.